packages/components/src/GuidanceBlock/GuidanceBlock.module.css

```diff
@@ -34,60 +12,67 @@ $scene-illustration-size: 300px;
 
 .banner {
   --border-width: var(--border-width-1);
+  --banner-padding: var(--spacing-lg);
```

Comment:
Probably worth updating the spacing variables to the current ones

Reply:
I've done it for all of them except for this one, because there's no equivalent to `lg`